shared/common-adapters/markdown/index.tsx

```diff
@@ -387,15 +388,23 @@ const noRules: {[type: string]: SimpleMarkdown.ParserRule} = {
 const noMarkdownParser = SimpleMarkdown.parserFor(noRules)
 
 const isAllEmoji = (ast: Array<SimpleMarkdown.SingleASTNode>) => {
-  const trimmed = ast.filter(n => n.type !== 'newline')
-  // Only 1 paragraph
-  // @ts-ignore
-  if (trimmed.length === 1 && trimmed[0]?.content?.some) {
-    // Is something in the content not an emoji?
-    // @ts-ignore
-    return !trimmed[0]?.content?.some(n => n.type !== 'emoji' && n.type !== 'newline')
+  let emojiLine = 0
+  for (const node of ast) {
```

Comment:
instead of doing a map / copy / etc just to see if we have one just iterate and keep track and bail early